src/backend/aspen/workflows/import_locations/save.py

```diff
@@ -35,21 +39,42 @@ def save():
         )
         session.execute(gisaid_locations_insert)
 
-        # Make sure we have equivalent rows for region/country/division but with null locations
-        except_select = (
+        # Insert an entry with a null location for every distinct Region/Country/Division combination
+        # Doing a double-equals comparison is critical for the statement to compile to the intended SQL,
+        # so we tell flake8 to ignore rule E711
+        existing_null_location_select = (
             sa.select(Location.region, Location.country, Location.division)
-            .where(Location.location is None)
+            .where(Location.location == None)  # noqa: E711
             .distinct()
         )
-        null_locations_select = (
-            sa.select(Location.region, Location.country, Location.division)
-            .distinct()
-            .except_(except_select)
+        existing_null_locations = set(
+            session.execute(existing_null_location_select).all()
         )
-        null_locations_insert = Location.__table__.insert().from_select(
-            ["region", "country", "division"], null_locations_select
+
+        gisaid_null_locations_select = sa.select(
```

Comment:
Piggy-backing on my above comment, if we want broader areas than null-Location, we'd need to make an additional select for each "zoomed out" area level.